config.con

```diff
@@ -1,5 +1,3 @@
 host "[::1]"
 port 8994
-sources {
-  default "https://github.com/DFabric/packages-source/tarball/master"
-}
+source "https://github.com/DFabric/packages-source/tarball/master"
```

Comment:
the last one was correct, this change is in DPPM@master

Reply:
ok thanks I was wondering which of those it was

Reply:
I changed it back to `sources {...}` and even after running `shards update` I got an exception -- `Unhandled exception: Unexpected token: EndHash (Exception)`

Reply:
Use this one https://github.com/DFabric/dppm/blob/master/config.con.
I also made a new commit to bump to Crystal 0.28.0, which is now in Alpine

Reply:
Thank you!